.github/workflows/go.yml

```diff
@@ -51,6 +51,11 @@ jobs:
 
       - name: Setup Go
         uses: actions/setup-go@v2
+        with:
+          go-version: 1.14
+
+      - name: set GOVERSION
+        run: echo "::set-env name=GOVERSION::$(go version | sed -r 's/go version go(.*)\ .*/\1/')"
```

Comment:
https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

Reply:
e.g. go1.14.8
![image](https://user-images.githubusercontent.com/23391543/92098551-5c172a00-ee14-11ea-9e84-fc3a2ee5236c.png)